gslib/tests/test_shim_util.py

```diff
@@ -1110,9 +1111,24 @@ def test_gcs_json_endpoint_translation_usees_default_version_v1(self):
       self.assertEqual(
           env_vars, {
               'CLOUDSDK_API_ENDPOINT_OVERRIDES_STORAGE':
-                  'https://foo_host:1234/storage/v1'
+                  'https://foo_host:1234/storage/v1/'
           })
 
+  def test_gcs_json_endpoint_translation_should_start_with_https(self):
```

Comment:
Isn't the test above taking care of ensuring `https` is present? Do we really need this additional test?

Reply:
Yeah, implicitly is testing that, I should probably delete this and add a comment?, I wanted to be clear that the `https` string must be present at the beginning of the string when the translation is made.

Reply:
I see. Yeah, a comment should be enough. We can skip the extra test.